AlfrescoApp/View Controllers/Sync View Controller/Data Source/SyncCollectionViewDataSource.m

```diff
@@ -95,4 +101,19 @@ - (void)setupDataSourceCollection:(id<NSFastEnumeration>)collection
     });
 }
 
+#pragma mark - RealmSyncManagerSyncDisabledDelegate methods
+- (void)syncFeatureStatusChanged:(BOOL)isSyncOn
+{
+    if(!isSyncOn)
+    {
+        [self.token stop];
+        [self setupDataSourceCollection:nil];
+    }
+    else
+    {
+        [self.token stop];
```

Comment:
As we try to respect DRY principle, please move `[self.token stop];` outside the if-else statement.